R/UK.R

```diff
@@ -0,0 +1,519 @@
+#' United Kingdom Class for downloading, cleaning and processing notification
+#' data.
+#'
+#' @description Country specific information for downloading, cleaning
```

Comment:
please provide some information on the data present (at least as much in previous docs)